### PR TITLE
Rename "authenticated" to "verified"

### LINKED ISSUE #44: Is "Authenticated" the right term for the mode?
I don't think that you can consider a resolver to be "authenticated" if all you have is an IP cert for an IP address that you found in an RA/DHCP packet.  That's no better than something that you found on a USB stick in a Walmart car park.

This is a policy statement: a client chooses to use a secured protocol for talking to a discovered resolver on the basis that it couldn't be worse than using an unsecured protocol.  This document claims that ownership over the IP address is strictly better.  That only assumes that the PKI you use to verify that ownership is trustworthy.  Thus, the secured protocol is OK to use.

_Originally posted by @martinthomson in https://github.com/ietf-wg-add/draft-ietf-add-ddr/pull/43#discussion_r741627771_

### PR DESCRIPTION
**Pull request by tfpauly**
Closes #44

### CODE REVIEW

File: draft-ietf-add-ddr.md [hunk 1]
```diff
@@ -225,12 +225,12 @@ Details of such policy are out of scope of this document. Clients SHOULD NOT
...
 
-Authenticated Discovery is a mechanism that allows automatic use of a
+Validated Discovery is a mechanism that allows automatic use of a
 Designated Resolver that supports DNS encryption that performs a TLS handshake.
 
-In order to be considered an authenticated Designated Resolver, the
+In order to be considered a validated Designated Resolver, the
 TLS certificate presented by the Designated Resolver MUST contain the IP address
 of the designating Unencrypted Resolver within the SubjectAlternativeName certificate
 field. If the certificate can be validated, the client SHOULD use the discovered
```
Comment (martinthomson) [inline draft-ietf-add-ddr.md:236]: The use of validated here might be worth looking at.  I don't know if you have other uses.  RFC 6125 uses "verified", so maybe that is better.

**Review by martinthomson — COMMENTED** [review]
This is an improvement.  Watch for existing uses of "validated" though.

File: draft-ietf-add-ddr.md (hunk 1 above)
Comment (tfpauly) [reply draft-ietf-add-ddr.md:236]: Yeah, verified can work too. Let's try that out.

Comment (tfpauly) [reply draft-ietf-add-ddr.md:236]: Okay, changed to verified—I had been debating between validated and verified to begin with.

File: draft-ietf-add-ddr.md [hunk 2]
```diff
@@ -388,7 +388,7 @@ where the Unencrypted Resolver and Designated Resolver have the same IP address.
 
 The constraints on validation of Designated Resolvers specified here apply specifically
 to the automatic discovery mechanisms defined in this document, which are
-referred to as Authenticated Discovery and Opportunistic Discovery. Clients
+referred to as Verified Discovery and Opportunistic Discovery. Clients
 MAY use some other mechanism to validate and use Designated Resolvers discovered
```
Comment (mstojens) [inline draft-ietf-add-ddr.md:392]: Where this is talking about a mechanism used in lieu of DDR verification, maybe it should also use the word "verify"? Not all existing uses of "validate" match that expectation, but this paragraph's instances seem to do so.

**Review by mstojens — APPROVED** [review]

File: draft-ietf-add-ddr.md (hunk 2 above)
Comment (tfpauly) [reply draft-ietf-add-ddr.md]: Good call, fixed

**Pull request merged**